src/pages/guides/getting-started/deployment.mdx

```diff
@@ -2,42 +2,29 @@ export const description = 'How to deploy Nitric applications'
 
 # Deployment
 
-## Managed Nitric with Deploy
+Nitric can automatically deploy your application to a supported cloud provider. As discussed in the [concepts](/guides/getting-started/concepts) page, Nitric has three standard `providers`, one each for AWS, Microsoft Azure and Google Cloud. If none of those fit your needs you're also able to [build a custom provider](/reference/providers/custom/building-custom-provider) to target the host you prefer with your own tools and configuration.
 
-[Nitric Deploy](https://deploy.nitric.io) is the fastest way to deploy your Nitric application with no configuration.
+The Nitric CLI command to perform deployments is `nitric up`. However, before your first deployment you'll need to create a `stack`, stacks represent deployment targets. If you haven't created a stack before we recommend reading the steps in our [quickstart guide](/guides/getting-started/quickstart#deploying-the-app).
```

Comment:
```suggestion
The Nitric CLI command to perform deployments is `nitric up`. However, before your first deployment you'll need to create a `stack`, stacks represent deployment targets. If you haven't created a stack before we recommend reading the steps in our [quick-start guide](/guides/getting-started/quickstart#deploying-the-app).
```

Reply:
Not sure that change is needed the word 'quickstart' is pretty common - e.g. https://docs.github.com/en/get-started/quickstart